fenics_concrete/__init__.py

```diff
@@ -9,4 +9,4 @@
 from fenics_concrete.helpers import Parameters
 from fenics_concrete.material_problems.concrete_thix_mechanical import ConcreteThixMechanical
 from fenics_concrete.experimental_setups.concrete_cube_uniaxial import ConcreteCubeUniaxialExperiment
-
+from fenics_concrete.experimental_setups.AMconcrete_Layer import MultipleLayers2DExperiment
```

Comment:
Just a thought: are you planning on implementing multiple experiments in this file?
otherwise I would suggest to try to match the name of the Class with that of the file, just to keep it more organized.

Reply:
sounds good. I would suggest that we then try to keep the names matching and you either then create a new class for 3D or if possible implement the change as an input parameter within this one